extensions/2.0/Vendor/EXT_feature_metadata/1.0.0/schema/gltf.EXT_feature_metadata.schema.json

```diff
@@ -35,11 +35,5 @@
     },
     "extensions": {},
     "extras": {}
-  },
-  "not": {
-    "required": [
-      "schema",
-      "schemaUri"
-    ]
```

Comment:
This is to ensure that `schema` and `schemaUri` are mutually exclusive and was copied from `camera.schema.json` in the glTF spec:

https://github.com/KhronosGroup/glTF/blob/c6a5521ebd1469aba076949a97f150bf3591b63f/specification/2.0/schema/camera.schema.json#L36-L38

I wonder if there's a better way to do this.